css-box-model/styles.css

```diff
@@ -0,0 +1,10 @@
+button.language {
+  font-size: 24px;
+  background-color: lightgray;
+
+  /* your code here 👇 */
+  margin-left: 12px;
+  margin-right: 12px;
+  padding: 6px, 16px;
```

Comment:
This should be a space, not a comma. It is a shorthand property, not a fallback value